### PR TITLE
NR-155179 Added Gcp Dahsboards

### PR DESCRIPTION
**Pull request by rajut-xrg**
# Summary
Added New dashboards for GCP

1. GCP Cloud Run
2. GCP Cloud SQL
3. GCP Dataproc
4. GCP Application Load Balancer
5. GCP Redis
6. GCP VPC Access Connector
7. GCP AppEngine
8. GCP BigQuery
9. GCP BigTable
10. GCP Cloud PUB/SUB
11.  GCP Cloud storage
12. GCP Cloud Function
13. GCP Data flow
14. GCP Data store
15. GCP Firebase database
16. GCP firebase hosting

There are still more to addign the dashbaord, which will be created a separate PR

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [ ] Did you include a Data source and Documentation reference?
- [ NA] Are all documentation links publicly accessible?
- [ NA] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### CODE REVIEW

**Comment by rajut-xrg** [comment]
Hi Team, could anybody start looking into this PR. Its been waiting from long time

**Comment by tmarchand-nr** [comment]
Hello -- do you have an estimate about when this PR can be reviewed?

File: dashboards/gcp-colud-functions/gcp-cloud-functions.json [hunk 1]
```diff
@@ -0,0 +1,207 @@
+{
+  "name": "GCP Cloud functions",
+  "description": null,
+  "permissions": "PUBLIC_READ_WRITE",
```
Comment (aswanson-nr) [inline dashboards/gcp-colud-functions/gcp-cloud-functions.json:4]: Just need to remove this line! We add a generic permission during installation.
```suggestion
```

**Review by aswanson-nr — COMMENTED** [review]
These dashboards look good to me! Just so you're aware, these will not be published as is, since they are not connected to a quickstart. That being said, we can move forward and merge them into the repository and they will be published as they are added to quickstarts. I also noticed one small change needed for the `gcp-cloud-functions` dashboard.

File: quickstarts/gcp/gcp-composer/config.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+id:
```
Comment (sarahkitten) [inline quickstarts/gcp/gcp-composer/config.yml:1]: Let's remove this line (the ID gets automatically generated)

File: quickstarts/gcp/gcp-memcache/config.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
+id:
```
Comment (sarahkitten) [inline quickstarts/gcp/gcp-memcache/config.yml:1]: Let's remove this line

**Review by sarahkitten — CHANGES_REQUESTED** [review]

**Comment by tmarchand-nr** [comment]
Keep open!

**Comment by sarahkitten** [comment]
@tmarchand-nr can you address the requested changes here

**Comment by sarahkitten** [comment]
closing as this is already addressed by https://github.com/newrelic/newrelic-quickstarts/pull/2262

**Pull request closed without merging**